docs/modules/player-lifecycle/basic-player-creation.md

```diff
@@ -0,0 +1,94 @@
+<%(TOC)%>
+# Implementing basic player creation
+
+<%(Callout message="
+Before reading this document, make sure you have read the following documentation on:
+
+* [Player Lifecycle Feature Module]({{urlRoot}}/modules/player-lifecycle/overview)
+* [Creating entity templates]({{urlRoot}}/reference/concepts/entity-templates)
+* [Workers in the GDK]({{urlRoot}}/reference/concepts/worker)
+")%>
+
+By default, the module automatically sends a player creation request as soon as the client-worker instance connects to SpatialOS. This spawns a [SpatialOS entity]({{urlRoot}}/reference/glossary#spatialos-entity) representing a player.
+
+To do this, you need to:
+
+1. Set up your worker connector.
+1. Define the entity template for your player entities.
+1. Configure what entity template to use for player creation.
+
+## Set up your worker connector
+
+After setting up the module, you must enable it by adding the necessary systems to your workers. This is done in the `HandleWorkerConnectionEstablished()` method of your [`WorkerConnector`]({{urlRoot}}/reference/workflows/monobehaviour/creating-workers), by using the following code snippets:
+
+1. On a [client-worker]({{urlRoot}}/reference/glossary#client-worker): `PlayerLifecycleHelper.AddClientSystems(Worker.World)`
+1. On a [server-worker]({{urlRoot}}/reference/glossary#server-worker): `PlayerLifecycleHelper.AddServerSystems(Worker.World)`
+
+To change this behaviour, read the documentation on [custom player creation]({{urlRoot}}/modules/player-lifecycle/custom-player-creation).
+
+## Define the player entity template
+
+The module takes care of spawning the player entity as soon as a client-worker connects. To enable this behaviour, the server-worker responsible for handling requests to spawn player entities needs to know which [entity template]({{urlRoot}}/reference/concepts/entity-templates) to use when sending the entity creation request to the [SpatialOS Runtime]({{urlRoot}}/reference/glossary#spatialos-runtime).
```

Comment:
~The module takes care of spawning the player entity as soon as a client-worker connects. To enable this behaviour,~